src/utils/isPlainObject.js

```diff
@@ -0,0 +1,3 @@
+export default function isPlainObject(obj) {
+  return obj ? typeof obj === 'object' && Object.getPrototypeOf(obj) === Object.prototype : false;
```

Comment:
What I want from `isPlainObject`:

* no to arrays
* no to null or undefined
* no to ImmutableJS records

I don't really care for edge cases like `Object.create(null)`.